TKGame/BackEnd/EntityManager.cs

```diff
@@ -96,8 +103,11 @@ public static void Update(GameTime gameTime)
             // Will need to do this for all unique entity lists, i.e. enemies, projectiles, etc.
             entities = entities.Where(x => !x.IsExpired).ToList();
 
+            weaponAttack.Update(entities[0]);
```

Comment:
this is definitely something that should be called when the player is updated

Reply:
again, needs access to the entities list to make the graphics work